OData/src/System.Web.OData/OData/Routing/Conventions/AttributeRoutingConvention.cs

```diff
@@ -53,8 +53,14 @@ public AttributeRoutingConvention(IEdmModel model, HttpConfiguration configurati
                 throw Error.ArgumentNull("configuration");
             }
 
+            DefaultODataPathHandler odataPathHanlder = pathTemplateHandler as DefaultODataPathHandler;
+            if (odataPathHanlder != null)
+            {
+                odataPathHanlder.ResolverSetttings = configuration.GetResolverSettings();
```

Comment:
Does this require a dependency on ODataLib 6.11?  Is it the issue is that ResolverSettings are not passed to pathTemplateHandler in existing code?